x-pack/platform/plugins/shared/fleet/common/types/rest_spec/data_stream.ts

```diff
@@ -10,3 +10,11 @@ import type { DataStream } from '../models';
 export interface GetDataStreamsResponse {
   data_streams: DataStream[];
 }
+
+export interface DeprecatedILMPolicyCheckResponse {
```

Comment:
you could define the both the schema and the response in that file to avoid duplicating the type see https://github.com/elastic/kibana/blob/fix-fetching-package-from-es/x-pack/platform/plugins/shared/fleet/common/types/rest_spec/agent_policy.ts#L72-L74